docs/UserGuide.md

```diff
@@ -1016,3 +1015,40 @@ the acceptable range). Therefore, edit the data file only if you are confident t
 3. **The commands are not case-insensitive**. The remedy is to ensure that all commands are either capitalised or not capitalised.
 4. Current **error message for invalid value** of `MONTHLY_RENT` and `DEPOSIT` is not precise enough (we use the term `numbers`). Please follow [Parameter Constraints Table](#parameter-constraints-table) for valid values.
 5. If a **parameter value is too long**, it will not display fully in the application. The excess characters will be cut off and replace by `...`. You should not provide a value that is too long to avoid this issue.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Glossary
+
+* **Extraneous**: Not directly related connected with or related to something.
+
+* **Client**: A landlord whose properties are managed by you (a letting agent).
+
+* **Letting agent**: An individual that facilitates agreements between tenants and landlords to rent properties.
+
+* **Customer list**: The list of tenants who are renting a property.
+
+* **Rent due date**: The date ('dd' in `dd/mm/yyyy`) in each month when monthly rent is collected from tenants.
+    Ranges between `1` - `31`, without leading zeros.
+
```

Comment:
Client, customer list and rent due date should explain in "notes about this user guide".
Letting agent may not be needed as the reader for this user guide is already a letting agent.

Reply:
They are already included in "notes about this user guide". They are included in glossary for completion.